livekit-agents/livekit/agents/worker.py

```diff
@@ -305,7 +306,16 @@ def __init__(
         async def health_check(_: Any):
             return web.Response(text="OK")
 
+        async def worker(_: Any):
+            body = json.dumps({
+                "agent_name": self._opts.agent_name,
+                "worker_type": self._opts.worker_type,
```

Comment:
That's kind of dumb that our WorkerType is a redefinition of the protobuf JobType object (instead of a str..)

Let's do this instead:
```suggestion
                "worker_type": agent.JobType.Name(self._opts.worker_type.value),
```

Reply:
This will give JT_ROOM, JT_PUBLISHER

Reply:
ngl i didn't test this. i was hoping you would be so offended by my implementation you would rewrite it correctly 😆

Reply:
lol no, I'm offended by my own implementation lol